app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,79 @@
+
+.page-logo {
+  text-align: center;
+}
+
+.img {
+  // margin-top: -10px;
+  // display: block;
+  width: 250px;
+  height: 250px;
+  background-color: #cccccc;
+  background: url("http://s9.postimg.org/h1lnkqwzz/image.jpg");
+}
+
+// p, ol, ul, td {
+//   font-family: verdana, arial, helvetica, sans-serif;
+//   font-size: 13px;
+//   line-height: 18px;
+// }
+//
+// pre {
+//   background-color: #eee;
+//   padding: 10px;
+//   font-size: 11px;
+// }
+//
+// a {
+//   color: #000;
+//
+//   &:visited {
+//     color: #666;
+//   }
+//
+//   &:hover {
+//     color: #fff;
+//     background-color: #000;
+//   }
+// }
+
+// div {
+//   &.field, &.actions {
+//     margin-bottom: 10px;
+//   }
+// }
+
+#notice {
+  color: green;
+}
+
+.field_with_errors {
+  padding: 2px;
+  background-color: red;
+  display: table;
+}
+
+#error_explanation {
```

Comment:
Avoid using id selectors<br>Selector `error_explanation` should be written in lowercase with hyphens